cluster.c

```diff
@@ -124,7 +124,7 @@ void addShardGroupFromArgs(RedisRaftCtx *rr, RedisModuleCtx *ctx, RedisModuleStr
         memcpy(sg_nodes[i].node_id, str, len);
         sg_nodes[i].node_id[len] = '\0';
 
-        str = RedisModule_StringPtrLen(argv[3 + i], &len);
+        str = RedisModule_StringPtrLen(argv[3+(i*2)], &len);
```

Comment:
Maybe we should clean up this number fiddling code a bit and avoid the magic numbers? Since it already introduced a bug once, it may do so again.

Reply:
Good idea, adding that